cmd/draft/create_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"testing"
 
 	"github.com/Azure/draft/pkg/draft/draftpath"
+	helpers "github.com/Azure/draft/pkg/testing"
```

Comment:
just a thought... Would it be better to move the CopyTree function to `pkg/testing/helpers` if we're always aliasing this package to `helpers`?